src/components/InputArea/index.js

```diff
@@ -1,26 +1,24 @@
 // @ts-check
-//import React from 'react';
+import React from 'react';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import { compose, setPropTypes } from 'recompose';
 
 import * as messageActions from '../../actions/messages';
 import { getLatestRemote } from '../../utils/helpers';
 
-const mapStateToProps = ({ messages }) => {
+const mapStateToProps = ({ messages, config }) => {
     let latestMessage = getLatestRemote(messages.messages);
     return {
         inputExpected: latestMessage && latestMessage.input_expected,
-        buttons: latestMessage && latestMessage.buttons
+        buttons: config.menu.buttons
     };
 };
 
 const mapDispatchToProps = dispatch => ({
-    submitHandler: text => {
+    submitHandler: payload => {
         dispatch(
-            messageActions.messageSend({
```

Comment:
@chrislombaard I assume the payload is an object that includes one or more of the following keys: postback, text, type

Reply:
Yes it is, that fine? @aleciafb

Reply:
Yes thanks!